### PR TITLE
fixing JSX type  module for React 19 types

### PR DESCRIPTION
**Pull request by Abse2001**
@seveibar is this the right approach I couldn't test it on snippet and after updating react types and fixing the type error I couldn't retest it even if I discard all changes I made

### CODE REVIEW

**Review by seveibar — COMMENTED** [review]
This is the right approach, but if you look at that react-three PR they declare 3 different modules (eqch one is relevant for different tsconfig modes)

You cant test locally? Have you tried cmd shift p then "restart ts server"?

**Comment by Abse2001** [comment]
@seveibar on core it looks like it works no more errors but I cant use yalc and overwrite the url in snippet ig

File: lib/fiber/intrinsic-jsx.ts [hunk 1]
```diff
@@ -1,53 +1,60 @@
 import type * as Props from "@tscircuit/props"
 
-declare global {
+export interface TScircuitElements {
```
Comment (seveibar) [inline lib/fiber/intrinsic-jsx.ts:3]: Casing, so we use google style single word casing and tscircuit we generally consider a single word
```suggestion
export interface TscircuitElements {
```

**Review by seveibar — APPROVED** [review]

**Comment by seveibar** [comment]
Yea with ATA testing is extremely difficult

**Pull request merged**